client/src/components/MainPage/TranslationsPage/DonateModal/index.jsx

```diff
@@ -22,10 +27,45 @@ class DonateModal extends Component {
     });
   };
 
+  onChange = (event) => {
+    const { name, value } = event.target;
+    this.setState({ [name]: value, translations: value });
+  };
+
+  onClick=(typeId) => {
```

Comment:
add space arount the = 
```suggestion
  onClick = (typeId) => {
```